_data/member_bios.yml

```diff
@@ -219,12 +219,117 @@ Tyler Nettesheim:
   photo_uri: '/assets/images/members/Nettesheim, Tyler.jpg'
   status: Student
 Wayne Chew:
-  connect_uri: 'https://www.linkedin.com/mynetwork/invite-sent/waynechew/?isSendInvite=true'
+  connect_uri: 'https://www.linkedin.com/in/waynechew/'
   name: Wayne Chew
   photo_uri: '/assets/images/members/Chew, Wayne.jpg'
   status: 'Software Engineer, Machine Learning, Data Science'
 Yunhe Liu:
   connect_uri: 'https://www.linkedin.com/in/yunhe-amos-liu-96b34068/'
   name: Yunhe 'Amos' Liu
   photo_uri: null
-  status: Studies at Cornell University
+  status: Studies at Cornell
+Ethan Amundson:
+  connect_uri: 'https://www.linkedin.com/in/ethanamundson'
+  name: Ethan Amundson
+  photo_uri: null
+  status: Vice President
+Wes Koerner:
+  connect_uri: 'https://www.linkedin.com/in/wesley-koerner-245471157/'
+  name: Wes Koerner
+  photo_uri: null
+  status: null
+John "Mack" Zern:
+  connect_uri: 'https://www.linkedin.com/in/john-zern-5911a8170/'
+  name: John "Mack" Zern
+  photo_uri: null
+  status: null
+Graham Mendola:
+  connect_uri: 'https://www.linkedin.com/in/graham-mendola-781b68167/'
+  name: Graham Mendola
+  photo_uri: null
+  status: null
+Bradley Mickelson:
+  connect_uri: 'https://www.linkedin.com/in/bradley-mickelson-0919a415b/'
+  name: Bradley mickelson
+  photo_uri: null
+  status: null
+Bryce Belgarde:
+  connect_uri: null
+  name: Bryce Belgarde
+  photo_uri: null
+  status: null
+Kiara Mutschler:
+  connect_uri: 'https://www.linkedin.com/in/kiara-mutschler/'
+  name: Kiara Mutschler
+  photo_uri: '/assets/images/members/Mutschler, Kiara.png'
+  status: Junior Studying Computer Science, Digital Studies Certificate
```

Comment:
Hmm I'm starting to think we should have two fields: 1 for status as displayed here, and one for KHK position like 'Pledge Trainer'. Do you think you could manage this?

Reply:
This is a great idea. I've added https://github.com/KappaEtaKappa/delta.khk.org/issues/16